docs/topics/testing/tools.txt

```diff
@@ -1566,6 +1566,38 @@ your test suite.
 
     Output in case of error can be customized with the ``msg`` argument.
 
+.. method:: SimpleTestCase.assertSignalSent(signal, required_kwargs=None)
+
```

Comment:
Please add `.. versionadded:: 1.11` to the new methods as well as release notes in `docs/releases/1.11.txt`.